### ISSUE TITLE
Bad rendering quality on Windows 10 mobile

### ISSUE THREAD

**Issue by crazyhappygame** [cocos2d/cocos2d-x#15901]
Please run cpp-test on Windows 10 mobile and Android and compare graphic.
It works fine (on both devices) but quality of rendered app is lower on Windows Phone.

I released (cocos2dx-3.10) exactly the same apps on Google Play and Windows Store. 
1. Google Play (https://play.google.com/store/apps/details?id=com.crazyhappygame.coloringpagespuzzleforkids).
2. Windows 10: (https://www.microsoft.com/store/apps/9nblggh4m297)

It seems like there is some issue with scaling/smoothing on WindowsPhone.
Both application uses: glview->setDesignResolutionSize(size.width, size.height, ResolutionPolicy::EXACT_FIT);

In attachment screen shots from devices: Android.png and WindowsPhone.png.
Android.png looks much better that WindowsPhone.png. How can I fix it?
Android: 720x1280
![android](https://cloud.githubusercontent.com/assets/13263288/16236923/61e5b25c-37da-11e6-98aa-1b20a6f774c0.png)
Windows: 768x1280
![windowsphone](https://cloud.githubusercontent.com/assets/13263288/16236931/698df532-37da-11e6-8bdf-ec4a6ae24cc9.png)

**Comment by ishmar** [comment]
Yeah, got same issue.  Don't have my phone now to capture screens but remember having it.

With original code it was easy to notice the ugly graphics (first I blamed 480x800 of Lumia 435). Later, I found that as I already had my designResolution(), removing the call to 
director->setContentScaleFactor() my game was still working, performance on Lumia 435 improved a bit, and graphics, while not being the same than on Android, looked better than before.

So I basically 'ignore' the hints given on [cocos 2d-x multi resolution tutorial](http://www.cocos2d-x.org/wiki/Multi_resolution_support) and just call 

glview->setDesignResolutionSize(designSize.width, designSize.height, ResolutionPolicy::EXACT_FIT);

After rereading the [detailed explanation](http://cocos2d-x.org/wiki/Detailed_explanation_of_Cocos2d-x_Multi-resolution_adaptation), it still seems a must to call setContentScaleFactor() but I would advice you to give it a try removing it and sharing results too.

**Comment by crazyhappygame** [comment]
@stammen could you help me?

**Comment by michael-ordering** [comment]
Are there any workround for solve this problem? I faced same thing on porting from Android to UWP.
How can I solve this?

**Comment by crazyhappygame** [comment]
No workaround so far ...

**Comment by ishmar** [comment]
Sadly, One of my top interest about using cocos2d-x was the possibility to develop for all platforms (android, ios AND wp), more than a year after, I've stopped all efforts for WP due the lack of support for basic stuff such as a fast app startup (blame ANGLE) and my schedule doesn't include update my WP game (yeah, it's published for WP8.1 and UWP win10/win10mobile) as there is no answer from development team.

My feeling is that WP port is (almost) abandoned so....

**Comment by dzpt** [comment]
I'm facing the same while scaling big image to small on iOS.
Have you solved this trouble?
